### PR TITLE
Fix/search bar usability

### PR DESCRIPTION
**Pull request by liuvasconcelos**
Fix search bar usability with cancel button.

### CODE REVIEW

File: StantUiIosLibrary/Classes/Bars/SearchBar/DefaultSearchBarDelegateExtension.swift [hunk 1]
```diff
@@ -8,19 +8,65 @@
...
+        if conditional {
+            isSearchEnable = false
+        } else {
+            isSearchEnable = true
+        }
+        
+        searchBar.showsCancelButton = isSearchEnable
     }
     
+    fileprivate func enableCancelButtonAndHidekeyboardOf(searchBar: UISearchBar) {
```
Comment (alexandrehcdc) [inline StantUiIosLibrary/Classes/Bars/SearchBar/DefaultSearchBarDelegateExtension.swift]: word **keyboard** should turn to capital case (enableCancelButtonAndHidekeyboardOf -> enableCancelButtonAndHideKeyboardOf)

File: StantUiIosLibrary/Classes/Bars/SearchBar/DefaultSearchBarDelegateExtension.swift [hunk 2]
```diff
@@ -8,19 +8,65 @@
...
+        isSearchEnable              = false
+        searchBar.text              = ""
+        searchOnTableView           = ""
+        searchBar.showsCancelButton = isSearchEnable
+        
+        searchBar.perform(#selector(self.resignFirstResponder), with: nil, afterDelay: 0.1)
+        delegate?.updateTableViewWith(search: "")
+    }
+    
+    fileprivate func checkCancelButtonVisibility(conditional: Bool, searchBar: UISearchBar) {
```
Comment (alexandrehcdc) [inline StantUiIosLibrary/Classes/Bars/SearchBar/DefaultSearchBarDelegateExtension.swift:55]: Are you using this variable isSearchEnable? Or you are just using to set to the showsCancelButton? If so, you could remove those conditions

File: StantUiIosLibrary/Classes/Bars/SearchBar/DefaultSearchBarDelegateExtension.swift (hunk 1 above)
Comment (liuvasconcelos) [reply StantUiIosLibrary/Classes/Bars/SearchBar/DefaultSearchBarDelegateExtension.swift]: Really. Done.

File: StantUiIosLibrary/Classes/Bars/SearchBar/DefaultSearchBarDelegateExtension.swift (hunk 2 above)
Comment (liuvasconcelos) [reply StantUiIosLibrary/Classes/Bars/SearchBar/DefaultSearchBarDelegateExtension.swift:55]: It is true. I did the adjustment. :)

**Review by alexandrehcdc — APPROVED** [review]

**Pull request merged**